scala/dllib/pom.xml

```diff
@@ -182,11 +182,12 @@
             <groupId>com.intel.analytics.bigdl.spark-version</groupId>
             <artifactId>${spark-version.project}</artifactId>
             <version>${project.version}</version>
-        </dependency>
-        <dependency>
+    	</dependency>
+	<dependency>
             <groupId>org.scalanlp</groupId>
             <artifactId>breeze_${scala.major.version}</artifactId>
-            <version>${breeze.version}</version>
+	    <version>${breeze.version}</version>
+	    <scope>${spark-scope}</scope>
```

Comment:
change to provided scope since breeze is only used in tests

Reply:
It seems also to be used in source code: 
[ERROR] /opt/work/jenkins/workspace/BigDL-PRVN-scalatest-optim-Spark-3.1/scala/dllib/src/main/scala/com/intel/analytics/bigdl/dllib/tensor/Tensor.scala:21: not found: object breeze
[ERROR] import breeze.linalg.{DenseMatrix => BrzDenseMatrix, DenseVector => BrzDenseVector}
http://10.112.231.51:18888/job/BigDL-PRVN-scalatest-optim-Spark-3.1/539/console

Reply:
try to exclude com.github.fommil.netlib:core in breeze for another try?